nilearn/datasets/func.py

```diff
@@ -2490,7 +2490,7 @@ def patch_openneuro_dataset(file_list):
             if old_pattern in name:
                 new_name = name.replace(old_pattern, new_pattern)
                 if not Path(new_name).exists():
-                    os.symlink(name, new_name)
```

Comment:
OK apparently this broke something
https://github.com/nilearn/nilearn/actions/runs/17798006641/job/50598390448#step:13:3986